.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "maven-build-files"]
```

Comment:
This submodule pulls in all the WPILib jars that we need. We use a local maven repo configured in the parent pom to allow artifact resolution from this dir.